tests/ui/borrowck/issue-17545.stderr

```diff
@@ -10,6 +10,9 @@ LL | |     ));
    | |      -- temporary value is freed at the end of this statement
    | |______|
    |        argument requires that borrow lasts for `'a`
+   |
+note: requirement that the value outlives `'a` introduced here
+  --> $SRC_DIR/core/src/ops/function.rs:LL:COL
```

Comment:
how come this doesn't actually print anything?

Reply:
because the span is in `std`, which don't get displayed when the sources aren't found (like when path remap is being used, which is done for all tests).